### ISSUE TITLE
Server Error (500)

### ISSUE THREAD

**Issue by Apehaenger** [djangopbx/djangopbx#12]
Need to reinstall my FusionPBX on a Debian12@Raspiberry Pi4 but didn't worked due to a lot install failures.
Looked for an alternative because FusionPBX start looking badly maintained without payed account and found yours.
Looks promising!!

Did a quick try of DjangoPBX via install script. Didn't noticed any issues during install.
After install, had to change http & https ports (because already used on my Raspi) and was able to successful log in.
Looks again promising.

But whenever I try i.e. a:
- Home -> [Switch] Dashboard 
- Status -> Registrations
- Status -> System Status

I'm getting a "Server Error (500)"

`systemctl status freeswitch` show that freeswitch looks running well.

Don't have any /var/log/django-pbx log, but do have a /home/django-pbx/pbx/general.log which logs:
```
TypeError: argument of type 'bool' is not iterable
ERROR Internal Server Error: /status/djangopbx/
Traceback (most recent call last):
  File "/home/django-pbx/envdpbx/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/django-pbx/envdpbx/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/django-pbx/envdpbx/lib/python3.11/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/django-pbx/pbx/status/views.py", line 90, in djangopbx
    if es.connect():
       ^^^^^^^^^^^^
  File "/home/django-pbx/pbx/pbx/fscmdabslayer.py", line 68, in connect
    if '-ERR' in resp:
       ^^^^^^^^^^^^^^
TypeError: argument of type 'bool' is not iterable
```
Any clue what I did wrong?!

**Comment by AdrianFretwell** [comment]
Hi There.
I had not intended to make DjangoPBX work on a RaspberryPI at this stage in it's development, but I am very glad it has installed and thank you very much for trying this, I think we should be able to get it working now you have got this far.

The issue appears to be in the event socket abstraction layer, I will investigate this.

The abstractiob layer allows the application to talk to FreeSWITCH either directly via the Event Socket or via AMQP (RabbitMQ), this is for larger installations like running a "Freeswitch farm".

Leave it with me...

**Comment by Apehaenger** [comment]
Hi Adrian!

Very happy to saw your quick response today, quite thanks as this place.
Also quite thanks supporting my early Raspberry Pi4 try as well as your detailed instructions howto test.

Here are my pleasing results:

Did a 
```
cd /home/django-pbx/pbx
./generic-update.sh
```
Which pulled your changes from git and asked some more which is answered with yes.

Immediately tried the previous "Server Error (500)" pages which are working now fine!!
Because I'm new to DjangoPBX, I have to dive into the details before saying more, but I found the "Status -> Switch Modules" page with most modules "running". Also a "Event Socket" was listed there as running.
Those which are listed as "Stopped" look okay to me (similar as in FusionPBX).

For sure I also tried your test suggestions:
Telnetting to localhost 8021 returned
```
Trying 127.0.0.1...
Connected to 127.0.0.1.
Escape character is '^]'.
Content-Type: auth/request
```
looked okay to me and
```
freeswitch@Raspi4-Keller> module_exists mod_event_socket
true
```
looks also quite cool!!

Will dive now into some open freeswitch config questions of mine, and go over trying to get DjangoPBX running.
At the moment, it looks to me like Raspberry Pi install is running quite well with your installer!!!

Quite thanks for your quick support and the great DjangoPBX work/possibility!
